### PR TITLE
 [MIG] account_invoice_transmit_method: Migration to 12.0

### PR DESCRIPTION
**Pull request by joshuajan**

### CODE REVIEW

File: account_invoice_transmit_method/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (Nikul-Chaudhary) [inline account_invoice_transmit_method/__init__.py]: @joshuajan no need, Remove it

File: account_invoice_transmit_method/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (Nikul-Chaudhary) [inline account_invoice_transmit_method/tests/__init__.py]: same here

**Review by Nikul-Chaudhary — COMMENTED** [review]
Minor Change

**Review by TDu — APPROVED** [review]
Quickly tested seems to work well

**Review by TDu — APPROVED** [review]
LGTM works well, could it be merged ?
The title of the PR may not be to the norm needs [12.0][MIG]

File: account_invoice_transmit_method/models/partner.py [hunk 3]
```diff
@@ -11,14 +10,14 @@ class ResPartner(models.Model):
 
     customer_invoice_transmit_method_id = fields.Many2one(
         'transmit.method', string='Customer Invoice Transmission Method',
-        company_dependant=True, track_visibility='onchange',
```
Comment (pedrobaeza) [inline account_invoice_transmit_method/models/partner.py]: Why are you removing this? You must keep it.

File: account_invoice_transmit_method/models/partner.py [hunk 4]
```diff
@@ -11,14 +10,14 @@ class ResPartner(models.Model):
...
+        track_visibility='onchange',
         domain=[('customer_ok', '=', True)], ondelete='restrict')
     customer_invoice_transmit_method_code = fields.Char(
         related='customer_invoice_transmit_method_id.code',
         string='Customer Invoice Transmission Method Code', readonly=True)
     supplier_invoice_transmit_method_id = fields.Many2one(
         'transmit.method', string='Vendor Invoice Reception Method',
-        company_dependant=True, track_visibility='onchange',
```
Comment (pedrobaeza) [inline account_invoice_transmit_method/models/partner.py]: The same

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by TDu** [comment]
@pedrobaeza  Thanks for the review I made a PR to fix those on the specific branch of @shine-it. Hoping it will be merged quickly

**Comment by TDu** [comment]
As to move forward getting this one merged
I have opened a new PR with the fix for the requested changes.
https://github.com/OCA/account-invoicing/pull/545

**Pull request closed without merging**